rosdep/base.yaml

```diff
@@ -226,6 +226,9 @@ beep:
   fedora: [beep]
   gentoo: [app-misc/beep]
   ubuntu: [beep]
+benchmark:
+  ubuntu:
```

Comment:
Typically I'd like to see this for at least Debian and Fedora as well (assuming the package is available here).  Can you add them?  Thanks.

Reply:
Done.

Also change:
``` bash
ubuntu:
  bionic:
```
to simply
``` bash
ubuntu:
```

Since that seems to be less restrictive and easier to maintain for future version, if I'm understanding things right. :)